app/Filament/Pages/Events.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Filament\Pages;
+
+use Filament\Pages\Page;
+
+class Events extends Page
+{
+    protected static ?string $navigationIcon = 'heroicon-o-document-text';
```

Comment:
This will most likely change with feedback from product, but for the time being, can we make this 
```suggestion
    protected static ?string $navigationIcon = 'heroicon-o-bell-alert';
```